lib/smart_answer_partial_template_wrapper.rb

```diff
@@ -0,0 +1,65 @@
+module SmartAnswerPartialTemplateWrapper
+  include ActionView::Helpers::TagHelper
+
+  def render(context, options, block)
+    content = super(context, options, block)
+    return content unless valid_path_and_format?(context)
+
+    content_tag(
+      :div,
+      govspeak_to_html(content),
+      data: { debug_template_path: smart_answer_partial_path }
+    )
+  end
+
+private
+
+  def valid_path_and_format?(context)
+    smart_answer_partial_path? && html_format?(context)
+  end
+
+  def html_format?(context)
+    context.request&.format&.symbol.to_s == "html"
+  end
+
+  def govspeak_to_html(content)
+    Govspeak::Document.new(remove_leading_spaces(content))
+      .to_html
+      .strip
+      .html_safe
+  end
+
+  def remove_leading_spaces(content)
+    content.to_str.gsub(/^ +/, "")
+  end
```

Comment:
This seems to duplicate logic from `ErbRenderer#strip_leading_spaces`. Is this duplication really necessary. If so, you should explain why in the commit note. If not, it should be removed.

Reply:
I believe that this presents the opportunity to fine tune this..